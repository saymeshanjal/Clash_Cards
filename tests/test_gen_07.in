10
14 37 5
2 45 2
12 3 1
16 32 5
14 4 3
6 12 5
8 11 3
2 35 3
6 32 3
16 6 3