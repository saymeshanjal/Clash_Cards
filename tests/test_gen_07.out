132 3300 0
0 1 1
336 588 0
374 12496 0
120 3312 0
0 20 0
4 66 0
0 2 0
6 14 0
1077 11793 0
